app/models/article.rb

```diff
@@ -33,6 +34,7 @@ class Article < ActiveRecord::Base
   validates :title, presence: { message: "Please specify a title"}
   validates_associated :subjects
   validates :subjects, presence: { message: 'at least one subject is required'}
+  validates :summary, presence: { message: 'Please use the last field at the bottom of this form to summarize your edits to the article in 140 chars or less.'}
```

Comment:
Can someone see a summary for each version as it is submitted, or do they only see the latest? Seems like with summary as a direct field on article, we're limiting it to the latter. Let me know if this is off.